taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.forms.css

```diff
@@ -142,7 +142,8 @@ ul.checkboxselectmultiple {
 
 /* Button */
 
-.button-wrapper button {
+.form-button,
+.button-wrapper button[type="submit"] {
   @extend .c-button;
   @extend .c-button--primary;
 }
```

Comment:
**Explanation.** More specific selectors. (Judge my logic, please.)

**Background.** I want TACC form component classes, but APCD switched to manual forms faster than I could create styles for them, so we are re-using Django CMS form plugin classes.

**Reasons.**
I assume[^1] devs will:
- use `.form-button` for submit button, with or without `.button-wrapper`, so _style as primary_.
- use `.button-wrapper` for one or more buttons, so _only a submit button should be primary_.

[^1]: I should document[^2] these instead of assume usage. Even if I make TACC form component, these styles still exist for Django CMS form plugin forms.
[^2]: So far, the documentation I have is [CMS form creation guides](https://confluence.tacc.utexas.edu/x/4ZB9E). I could document Django CMS form styles during [FP-1828](https://jira.tacc.utexas.edu/browse/FP-1828).